### PR TITLE
eos-metrics-collector.exe style trivia

### PR DESCRIPTION
**Pull request by wjt**
I had a quick read of this script and applied some linters.

### CODE REVIEW

File: tools/eos-metrics-collector.exe [hunk 1]
```diff
@@ -48,12 +47,12 @@ class OfflineMetrics():
     def metrics_unit(self, operation):
         # TODO: the metrics daemon is also D-Bus activatable, so on stop we
         # should also do something to prevent it being started over D-Bus.
```
Comment (wjt) [inline tools/eos-metrics-collector.exe:49]: Yes, but, I believe it's activated via systemd so masking the systemd unit should DTRT.

File: tools/eos-metrics-collector.exe [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 import argparse
 import dbus
-import errno
```
Comment (manuq) [inline tools/eos-metrics-collector.exe:5]: :+1:

**Review by manuq — APPROVED** [review]
Looks good. There are some ways to move forward to keep the repository free of lint errors:
- Run black with a (pre-commit)[https://github.com/psf/black#version-control-integration] hook.
- Setup a python virtualenv with both black and pre-commit as dev requirements. So all devs will have the same environment. [pipenv](https://pypi.org/project/pipenv/), [fades](https://fades.readthedocs.io/en/stable/) and similar tools make this easy.
- Integrate black with the CI tool.

**Comment by wjt** [comment]
I support automation!

**Comment by jprvita** [comment]
In my defense, I brought flake8 with me along this journey (but obviously forgot about it on the last commit where I added `errno` back). I did not know about black.

I like the idea of having a standardized dev environment -- it saves me from having to get a refresher on python's best practices twice a year when I have to touch some python code again :)

**Pull request merged**